permissions/build.gradle

```diff
@@ -43,15 +43,19 @@ android {
 
 dependencies {
     implementation project(':strings')
+    implementation project(':androidshared')
 
     implementation Dependencies.androidx_core_ktx
     implementation Dependencies.androidx_fragment
     implementation Dependencies.android_material
+    implementation Dependencies.karumi_dexter
```

Comment:
We can probably remove this dependency from the `collect_app` `build.gradle` now.

Reply:
It was used in one test but now I got rid of it and it is really unused.

Reply:
It was used in one test but now I got rid of it and it is really unused.